src/cryptoadvance/specter/templates/includes/tx-row.html

```diff
@@ -106,10 +106,14 @@
             <span class="amount-price note hidden">()</span>
         </td>
         <td id="column-time" class="time"></td>
-        <td id="column-confirmations" >
-            <span class="confirmations"></span>
-            <button class="rbf btn optional hidden" style="width: 130px; float: right;" type="button">{{ _("Speed up") }}</button>
-            <button class="rbf-cancel danger btn optional hidden" style="width: 130px; float: right; margin-right: 10px;" type="button">{{ _("Cancel transaction") }}</button>
+        <td id="column-confirmations">
```

Comment:
Looks already really good, just played around in the firefox dev tools and those little changes appeared to be a nice fine-tuning, what do you think @b30wulffz?

<img width="428" alt="grafik" src="https://user-images.githubusercontent.com/70536101/165540805-ab675393-253f-4832-bc66-e94e9b65fd38.png">

Reply:
I tested this, both designs look nice to me. Can't decide.
![image](https://user-images.githubusercontent.com/31105595/165630472-89a72cbf-7560-4a50-909b-35e6d26952c7.png)


Also, I feel that we can avoid margin-right, until very much required. I get an OCD  xD, as it looks like we are disturbing the  central alignment of the div beside it.
We are doing this because last td in table for each row, is empty and thus it looks like a block before that is misaligned a bit.

But what if last td gets filled, then it will be a little misaligned.

Reply:
Also, spotted a bug, we need to supply a fixed width of confirmation "first" container, else button's positions will get disturbed if buttons are open for multiple transactions.